CHANGELOG/CHANGELOG-1.7.md

```diff
@@ -91,7 +91,7 @@ NA
 - Add config of cloudcore token refresh frequence ([#2796](https://github.com/kubeedge/kubeedge/pull/2796), [@leofang94](https://github.com/leofang94))
 - keadm: install CRDs corresponding to specific version ([#2803](https://github.com/kubeedge/kubeedge/pull/2803), [@daixiang0](https://github.com/daixiang0))
 - make customsiz labels available when restart ([#2839](https://github.com/kubeedge/kubeedge/pull/2839), [@ttlv](https://github.com/ttlv))
-
+- add router servicebus ([#2588](https://github.com/kubeedge/kubeedge/pull/2588), [@WintonChan](https://github.com/WintonChan))
```

Comment:
Would you like to add this to `What's new` section, and add some descriptions?

Reply:
ok